swarm/storage/localstore_test.go

```diff
@@ -0,0 +1,153 @@
+package storage
+
+import (
+	"io/ioutil"
+	"os"
+	"sync"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/contracts/ens"
+)
+
+var (
+	hashfunc = MakeHashFunc("SHA3")
+)
+
+// convenience generator for unique chunks
+type randomChunkGenerator struct {
+	data   []byte
+	hasher SwarmHash
+}
+
+func newRandomChunkGenerator(stem []byte) *randomChunkGenerator {
+	gen := &randomChunkGenerator{
+		data:   make([]byte, 8),
+		hasher: hashfunc(),
+	}
+	gen.data = append(gen.data, stem...)
+	return gen
+}
+
+func (self *randomChunkGenerator) newChunk() *Chunk {
+	self.hasher.Reset()
+	self.hasher.Write(self.data)
+	chunk := NewChunk(self.hasher.Sum(nil), nil)
+	chunk.SData = make([]byte, len(self.data))
+	copy(chunk.SData, self.data)
+	self.data[0]++
+	return chunk
+}
+
+// put to localstore and wait for stored channel
+// does not check delivery error state
+func putChunks(store *LocalStore, chunks ...*Chunk) {
```

Comment:
This should also be extracted and placed somewhere where we can reuse it, rather than a private function in the tests.

Reply:
Now or later PR? We should probably have a uniform way of placing helper functions in files? @nonsense